src/util/nidmap.c

```diff
@@ -422,6 +422,9 @@ int prte_util_decode_nidmap(pmix_data_buffer_t *buf)
         prte_rml_compute_routing_tree();
     }
 
+    /* update the routing tree */
+    prte_rml_compute_routing_tree();
```

Comment:
@rhc54 Should the additional `prte_rml_compute_routing_tree()` on line 422 be dropped, so not compute routing tree twice?

Reply:
yeah - no point in doing it twice!